Topic_1/my-app/src/test/java/com/mycompany/app/my_app/AppTest.java

```diff
@@ -0,0 +1,25 @@
+package com.mycompany.app.my_app;
+
+import junit.framework.TestCase;
+
+public class AppTest extends TestCase{
+	private static App app;
+	
+	public void setUp() {
+		app = new App(2,3);
+	}
+	
+	public void testMajor() {
+		int value = app.major();
+		assertNotSame(2, value);
+		assertEquals(3, value);
+    }
+	
+	public void testMinor() {
```

Comment:
It's better to use names with given something... when do something... then happend something... This make the code more readable.